packages/server-wallet/e2e-testing/test-data/load-data.json

```diff
@@ -1,10 +1,10 @@
 [
```

Comment:
One idea that might be fun to explore in future is the use of deterministic pseudo random numbers: for example, https://www.npmjs.com/package/prando. 

We could then have the entire load file be generated deterministically from e.g. some seed phrase, which would be pretty great for reproducibility (and would't require checking in the load file itself).

Reply:
> Standard question for autogenerated files (especially reasonably large ones): should this be checked in?
> 
> I understand that it contains _random_ output, so perhaps there is an argument for checking it in -- we want to be able to reproduce stress test runs, or run the same run during CI or something?

The main motivation for checking this is in was to have a sample "load-file". It's also used for the `start-all` script to get up and running quickly.

The file is **not** autogenerated so it generally won't change. However, it's being updated in this PR since I added new step types and modified existing types so I had to generate a new file to keep compatible with the changes. 

In the future, I was thinking that we may have a couple of "scenario" load files checked in. These could then be used for local or CI testing.

I think I should name the current file to `sample-load-file.json` so it's more obvious that it's a sample.

> One idea that might be fun to explore in future is the use of deterministic pseudo random numbers: for example, https://www.npmjs.com/package/prando.

I've also had that idea and would like to explore it in the future! It would be really nice for reproducing issues if you just have to share a seed.